File: bddtests/src/main/kotlin/mocking/tpp/models/TppAddress.kt
package mocking.tpp.models

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlAttribute

@XmlAccessorType(XmlAccessType.FIELD)
data class TppAddress(
        @XmlAttribute var address: String = "default address",
        @XmlAttribute var addressType: String = "default address type"
)

File: backendworker/NHSOnline.Backend.GpSystems/Suppliers/Tpp/Models/TppAddress.cs
using System.Xml.Serialization;

namespace NHSOnline.Backend.GpSystems.Suppliers.Tpp.Models
{
    public class TppAddress
    {
        [XmlAttribute(AttributeName="address")]
        public string Address { get; set; }
        [XmlAttribute(AttributeName="addressType")]
        public string AddressType { get; set; }
    }
}